shardingsphere-operator/pkg/reconcile/storagenode/aws/rdsinstance.go

```diff
@@ -116,11 +183,23 @@ func (c *RdsClient) DeleteInstance(ctx context.Context, node *v1alpha1.StorageNo
 	if err != nil {
 		return err
 	}
-	if ins == nil || ins.DBInstanceStatus == "deleting" {
+	if ins == nil || ins.DBInstanceStatus == v1alpha1.StorageNodeInstanceStatusDeleting {
 		return nil
 	}
 
-	instance.SetDeleteAutomateBackups(true)
-	instance.SetSkipFinalSnapshot(true)
+	var isBackup, isSkipFinalSnapshot bool
```

Comment:
use `isDeleteBackup` to replace `isBackup` can more clearly explain what you want do.

Reply:
done